AllAfrica.js

```diff
@@ -1,6 +1,6 @@
 {
 	"translatorID": "34B1E0EA-FD02-4069-BAE4-ED4D98674A5E",
-	"label": "allAfrica.com",
+	"label": "AllAfrica",
```

Comment:
The label should match the filename, otherwise Scaffolds tries to overwrite this every time.